dm/dm/ctl/master/config.go

```diff
@@ -331,6 +331,22 @@ func getSourceCfgs(cli *clientv3.Client) (map[string]*config.SourceConfig, error
 	return sourceCfgsMap, nil
 }
 
+// getRelayCfgs gets all relay cfgs.
+func getRelayCfgs(cli *clientv3.Client) (map[string]map[string]struct{}, error) {
+	relayCfgsMap, _, err := ha.GetAllRelayConfig(cli)
+	if err != nil {
+		return nil, err
+	}
+	// try to get all source cfgs before v6.2.0
+	if len(relayCfgsMap) == 0 {
+		relayCfgsMap, _, err = ha.GetAllRelayConfigBeforeV620(cli)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return relayCfgsMap, nil
+}
+
 func getAllCfgs(cli *clientv3.Client) (map[string]*config.SourceConfig, map[string]map[string]config.SubTaskConfig, map[string]map[string]struct{}, error) {
```

Comment:
just a question. `getAllCfgs` may be just used to export config, here change to support read from old keys is because we support export config when rolling update and there are both new and old version?

Reply:
Similar to `getSourceCfgs`. To support dumping relay configs for earlier dm clusters. We only try to dump old keys when there is no relay configs in new dm cluster.